app/scripts/metamask-controller.js

```diff
@@ -2205,6 +2205,13 @@ export default class MetamaskController extends EventEmitter {
     this.extension.runtime.onMessageExternal.addListener(onMessageReceived);
     // Fire a ping message to check if other extensions are running
     checkForMultipleVersionsRunning();
+
+    if (this.onboardingController.store.getState().completedOnboarding) {
+      // TODO: Delete when ready to remove `networkVersion` from provider object
+      this.deprecatedNetworkId = null;
```

Comment:
Could we move this to a function, so that we don't need to repeat it? e.g. something like `postOnboardingInitialization` maybe

Reply:
addressed in `7e532de6f6554a37aa625882d12652c8a054e36c`